### PR TITLE
Missing filter field should not 500

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?189016

@esoergel

### CODE REVIEW

File: corehq/apps/userreports/reports/data_source.py [hunk 1]
```diff
@@ -126,16 +127,17 @@ def column_warnings(self):
     def get_data(self):
         try:
             ret = super(ConfigurableReportDataSource, self).get_data()
-            for report_column in self.column_configs:
-                report_column.format_data(ret)
         except (
             ColumnNotFoundException,
```
Comment (nickpell) [inline corehq/apps/userreports/reports/data_source.py]: should look into updating sqlagg to consolidate ```ColumnNotFoundException``` and ```ColumnWithNameNotFoundException``` if they mean the same thing

Comment (esoergel) [inline corehq/apps/userreports/reports/data_source.py]: Seems like at the very least `ColumnWithNameNotFoundException` could subclass `ColumnNotFoundException`, or maybe they both have a common ancestor which would make sense to use here.

Comment (esoergel) [inline corehq/apps/userreports/reports/data_source.py]: I just checked, they don't have a common ancestor, and they're defined in different parts of SqlAgg, which seems confusing.

Comment (esoergel) [inline corehq/apps/userreports/reports/data_source.py]: Looks like I'm late to the party: https://github.com/dimagi/sql-agg/pull/25.  Are you gonna bump our sqlagg version in HQ and change this back?

Comment (nickpell) [inline corehq/apps/userreports/reports/data_source.py]: :+1:

**Pull request merged**